lib/api/controllers/securityController.js

```diff
@@ -643,25 +659,25 @@ function createOrReplaceProfile (profileRepository, request, opts) {
  * @returns {Promise.<*>}
  */
 function resetRoles (defaults, roleRepository) {
-  let promises = ['admin', 'default', 'anonymous'].map(id => {
-    let role = Object.assign(new Role(), {_id: id}, defaults[id]);
+  const promises = ['admin', 'default', 'anonymous'].map(id => {
+    const role = Object.assign(new Role(), {_id: id}, defaults[id]);
     return roleRepository.validateAndSaveRole(role);
   });
 
-  return Promise.all(promises);
+  return Bluebird.all(promises);
 }
 
 /**
  * @param {ProfileRepository} profileRepository
  * @returns {Promise.<*>}
  */
 function resetProfiles (profileRepository) {
-  let promises = ['admin', 'default', 'anonymous'].map(id => {
-    let profile = Object.assign(new Profile(), {_id: id, policies: [{roleId: id}]});
+  const promises = ['admin', 'default', 'anonymous'].map(id => {
+    const profile = Object.assign(new Profile(), {_id: id, policies: [{roleId: id}]});
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 86 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)